manifests/deployment.yml

```diff
@@ -29,6 +29,48 @@ spec:
           limits:
             cpu: 750m   
             memory: 2048M
+      - name: pgbouncer
+        image: uisacr.azurecr.io/pgbouncer:latest
+        ports:
+        - containerPort: 5432
+        resources:
+          requests:
+            cpu: 500m
+          limits:
+            cpu: 2048m   
+            memory: 4096M
+        env:
+            - name: NODE_ENV

```

Comment:
node env should not be needed for pgbouncer

Reply:
Addressed in follow-up commit